### ISSUE TITLE
Visualizer doesn't display anything on Linux

### ISSUE THREAD

**Issue by steve-r-west** [a85/POSTMan-Chrome-Extension#9039]
**Describe the bug**
Visualizer is empty when something is being visualized.

**To Reproduce**
1. Go to https://learning.postman.com/docs/sending-requests/visualizer/#rendering-html
2. Click on *Run in Postman*
3. Click on *Postman for Linux*
4. Execute Request in Postman
5. Click on *Body* -> *Visualize*

**Expected behavior**
You should see something

**Actual behavior**

Nothing is shown

**Screenshots**
![image](https://user-images.githubusercontent.com/49958178/92538514-b426a580-f1f3-11ea-8c98-15dce1c9c859.png)

Note if you don't call the visualizer, I instead see:

![image](https://user-images.githubusercontent.com/49958178/92538540-cb659300-f1f3-11ea-9d24-4f4277073a2c.png)

**App information (please complete the following information):**
 - App Type Chrome
 - Postman Version 7.32.0
 - OS: Linux / Arch.

**Additional context**
If I inspect the UI, and drill into the DOM, I can eventually change the <webview> that loads the context, as follows:

```
<div class="response-visualizer-viewer">Pre Webview Tag
<webview class="response-visualizer-webview-viewer__webview" id="visualizer_webview" src="file:///tmp/j0HbBApNx.html" webpreferences="nodeIntegration=no, javascript=yes" partition="visualizer" style="width:640px; height:480px"></webview>
Post Webview Tag
<iframe src="file:///tmp/j0HbBApNx.html">
</div>
</iframe></div>
```

This essentially forces the width and height on the webview, adds some text and then adds an iframe.

If I do this (and hover the webview) in the developer tools, I see.

![image](https://user-images.githubusercontent.com/49958178/92538390-63af4800-f1f3-11ea-9091-485db3fca95e.png)

That tells me that the webview is there, and the file is there (since the iframe has content), but something about the webview is not being rendered.

I didn't see anything of note in the Postman console, dev tool console, or anything.

**Comment by DannyDainton** [comment]
Hey @steve-r-west 

I've not been able to replicate this problem on my Linux machine - I see the table rendered in the visualize tab. 

In your final image, I can see that the table is rendered - Are you seeing that additional empty space each time you send the request? 

You can debug the visualizer, by right-clicking in the visualize tab and it will display the `Inspect visualization` option.

**Comment by steve-r-west** [comment]
Hi @DannyDainton , 

I have been able to reproduce this on two linux machines (both arch). This one running Postman 7.26.1.

> In your final image, I can see that the table is rendered - Are you seeing that additional empty space each time you send the request?

In the final image **I have hacked the** DOM using developer tools, I forced the webview to be a specific size. The table you see is _not_ postman, it's the iframe I added. _The first screenshot with the empty space is what I actually see_. As best I can tell when using this template stuff, postman renders a file on disk and then uses a `webview` to render it in the UI. The point of the final screenshot is thusly:

1) to show that the contents are rendered on disk properly.
2) to show that webview doesn't display anything even with a specific forced size
3) To show that that area is visible. 

I cannot "right click the visualize tab" at no point does a menu pop up . I can get to the DOM by going to View -> Developer -> Show Dev Tools.

**Comment by steve-r-west** [comment]
Hi @DannyDainton, I debugged this a bit more. This is entirely a problem with how my distro is building the postman package (using the one off the website, or the `pre-built` one works fine) I suspect it might have something to do with the version of Electron it is using maybe. 

I did find this issue, which says that webviews are disabled in newever versions of Electron, and it looks like it is using Electron 7 https://www.electronjs.org/docs/api/browser-window . Although that might be entirely irrelevant.

Feel free to close this.
